### PR TITLE
update component props

### PR DESCRIPTION
**Pull request by localyost3000**

### CODE REVIEW

File: dynaboard.yml [hunk 1]
```diff
@@ -129,22 +164,28 @@ nodes:
...
     properties:
       height: 80px
+      isEnabled: '{{true}}'
+      isVisible: '{{true}}'
       renderMode: markdown
+      showExternalIndicator: '{{false}}'
       styles: ''
       top: '5'
       value: >-
-        This getting started guide is here to help you learn the ins and outs
```
Comment (remyguercio) [inline dynaboard.yml:136]: This never got back propagated to the original template.

File: dynaboard.yml [hunk 2]
```diff
@@ -210,7 +254,7 @@ nodes:
               "https://dynaboard.app/_next/image?url=https%3A%2F%2Fapp-cdn.dynaboard.com%2Fportal-app%2F_next%2Fstatic%2Fmedia%2FtemplateUsersLight.d244dd42.png&w=640&q=75",
             templateTitleText: "User Admin Panel",
             templateDescText:
-              "See how to build a data-driven UI in Dynaboard involving the data table.",
```
Comment (remyguercio) [inline dynaboard.yml]: Same here

File: dynaboard.yml [hunk 3]
```diff
@@ -220,7 +264,7 @@ nodes:
               "https://dynaboard.app/_next/image?url=https%3A%2F%2Fapp-cdn.dynaboard.com%2Fportal-app%2F_next%2Fstatic%2Fmedia%2FtemplateGPTLight.eb1c3bb2.png&w=640&q=75",
             templateTitleText: "GPT-3 Playground",
             templateDescText:
-              "Get a head start on building your own GPT-3-powered application with a pre-built UI and included prompt examples.",
```
Comment (remyguercio) [inline dynaboard.yml]: Here

File: dynaboard.yml [hunk 4]
```diff
@@ -230,7 +274,7 @@ nodes:
               "https://dynaboard.app/_next/image?url=https%3A%2F%2Fapp-cdn.dynaboard.com%2Fportal-app%2F_next%2Fstatic%2Fmedia%2FtemplateDemosLight.5f70eaaa.png&w=640&q=75",
             templateTitleText: "Lightning Demos",
             templateDescText:
-              "Experience a complete application that leverages Dynaboard's built-in KV store for application storage.",
```
Comment (remyguercio) [inline dynaboard.yml]: Here

File: dynaboard.yml [hunk 5]
```diff
@@ -659,8 +785,8 @@ nodes:
         }
       top: '2'
       value: >-
-        So far, we’ve only returned a basic string value inside of a data
```
Comment (remyguercio) [inline dynaboard.yml]: Another

File: dynaboard.yml [hunk 6]
```diff
@@ -727,26 +856,32 @@ nodes:
...
+      height: '5'
+      isEnabled: '{{true}}'
+      isVisible: '{{true}}'
       renderMode: markdown
+      showExternalIndicator: '{{false}}'
       styles: ''
       top: '2'
       value: >-
         Below we have a small user account management example that includes a
-        data table, text input, and dropdown. With `Test` mode selected, choose
```
Comment (remyguercio) [inline dynaboard.yml]: No good deed goes unpunished?

File: dynaboard.yml [hunk 7]
```diff
@@ -1188,11 +1382,14 @@ nodes:
...
     properties:
       height: '4'
+      isEnabled: '{{true}}'
+      isVisible: '{{true}}'
       renderMode: markdown
+      showExternalIndicator: '{{false}}'
       styles: ''
       top: '11'
       value: >-
-        To see how it works, select the function in the bottom left corner of the
```
Comment (remyguercio) [inline dynaboard.yml:1195]: Last one

File: dynaboard.yml [hunk 8]
```diff
@@ -1366,17 +1593,21 @@ nodes:
           padding-x: 8px;
         }
       value: Fork →
+      showExternalIndicator: '{{false}}'
       fontSize: sm
       textAlign: right
       verticalAlign: bottom
       fontWeight: bold
-      color: "{{isHovered.value ? 'text.primary' : 'text.muted'}}"
+      color: '{{isHovered.value ? ''text.primary'' : ''text.muted''}}'
```
Comment (remyguercio) [inline dynaboard.yml]: @swimmadude66 should this have double single quotes?

File: dynaboard.yml [hunk 9]
```diff
@@ -66,25 +72,41 @@ nodes:
         - nodeID: 48d9fd9c-86a9-421f-9c3a-b7d595524ed1
           orderKey: ZzV
     properties:
+      footerProps:
+        element: ''
+      headerProps:
+        element: ''
+      includeInSitemap: '{{true}}'
       layoutStyle: VERTICAL_STACK
-      minWidth: 768px
```
Comment (remyguercio) [inline dynaboard.yml:70]: I'm not sure why the min-width changed? May have been me, but it should be tablet sized.

Comment (localyost3000) [reply dynaboard.yml:70]: I think the default changed, so its being dropped from export

File: dynaboard.yml (hunk 6 above)
Comment (localyost3000) [reply dynaboard.yml]: This one is not good grammar. It should be 
> With `Test` mode selected choose an account type, search for an account owner, or sort the data table.

the comma after selected is out of place, but the other one makes sense

File: dynaboard.yml (hunk 8 above)
Comment (localyost3000) [reply dynaboard.yml]: its just how the yaml got exported. its still valid, but I'll switch it to double quotes for readability

File: dynaboard.yml (hunk 6 above)
Comment (remyguercio) [reply dynaboard.yml]: "With Test mode selected" is a dependent clause, so it should be separated using a comma.

Comment (remyguercio) [reply dynaboard.yml]: There is an understood you beginning the following independent clause.

File: dynaboard.yml [hunk 10]
```diff
@@ -727,13 +857,16 @@ nodes:
...
+      isEnabled: '{{true}}'
+      isVisible: '{{true}}'
       renderMode: markdown
+      showExternalIndicator: '{{false}}'
       styles: ''
       top: '2'
       value: >-
         Below we have a small user account management example that includes a
-        data table, text input, and dropdown. With `Test` mode selected, choose
+        data table, text input, and dropdown. With `Test` mode selected choose
```
Comment (localyost3000) [inline dynaboard.yml]:
```suggestion
        data table, text input, and dropdown. With `Test` mode selected, choose
```

**Review by remyguercio — APPROVED** [review]
LGTM

**Pull request merged**